ores/features/azwiki.py

```diff
@@ -1,33 +1,208 @@
 from revscoring.features import (
-    added_badwords_ratio, added_misspellings_ratio, day_of_week_in_utc,
-    hour_of_day_in_utc, is_content_namespace, is_custom_comment,
-    is_section_comment, longest_repeated_char_added, longest_token_added,
-    numeric_chars_added, prev_words, proportion_of_markup_added,
-    proportion_of_numeric_added, proportion_of_symbolic_added,
-    proportion_of_uppercase_added, seconds_since_last_page_edit,
-    user_age_in_seconds, user_is_anon, user_is_bot, words_added, words_removed
+    diff, page, parent_revision, previous_user_revision, revision, user
 )
 from revscoring.features.modifiers import log
 
 damaging = [
```

Comment:
Indeed. We could even delete the language specific files in ores/features/ as generic.generic has the same feature sets for "damaging" and "good_faith".